__tests__/app.test.js

```diff
@@ -104,25 +101,52 @@ describe("GET/api/articles?topic=banana", () => {
 });
 
 describe("/api/articles/:article_id", () => {
-    
   describe("GET/api/articles/:article_id", () => {
-      test("200: return article with comment_count", () => {
+    test("200: return article with comment_count", () => {
+      return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body: article }) => {
+          expect(article).toEqual({
+            article_id: 1,
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: "2020-07-09T20:11:00.000Z",
+            votes: 100,
+            comment_count: 11,
+          });
+        });
+    });
+    describe("GET/api/articles/:article_id/comments", () => {
+      test("200: return comments with relevent article id", () => {
         return request(app)
-          .get("/api/articles/1")
+          .get("/api/articles/1/comments")
           .expect(200)
-          .then(({ body: article }) => {
-            expect(article).toEqual({
-              article_id: 1,
-              title: "Living in the shadow of a great man",
-              topic: "mitch",
-              author: "butter_bridge",
-              body: "I find this existence challenging",
-              created_at: "2020-07-09T20:11:00.000Z",
-              votes: 100,
-              comment_count: 11,
+          .then(({ body: comments }) => {
+            expect(comments).toHaveLength(11);
+            comments.forEach((comments) => {
+              expect(comments).toEqual(
+                expect.objectContaining({
+                  comment_id: expect.any(Number),
+                  author: expect.any(String),
+                  body: expect.any(String),
+                  created_at: expect.any(String),
+                  votes: expect.any(Number),
+                })
+              );
             });
           });
       });
+      test("400: return error", () => {
```

Comment:
Can you add a bit more description here, what kind of error, in response to what?
"/api/articles/1123123/comments" has an invalid article ID, of the correct datatype so needs to be a 404 rather than a 400.